packages/kbn-dependency-usage/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "name": "@kbn/dependency-usage",
+  "private": true,
+  "version": "1.0.0",
+  "license": "Elastic License 2.0 OR AGPL-3.0-only OR SSPL-1.0",
+  "type": "module",
```

Comment:
> [!NOTE]  
> `dependency-cruiser` is a esm only package, so I need to adjust `tsconfig` to `"module": "esnext"` and have a "type": "module" for it in `package.json`, otherwise it was failing with 
```
No "exports" main defined in /node_modules/dependency-cruiser/package.json
```